src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="menu header__menu">
+        <a href="#" class="menu__logo">
+          <img
+          src="./images/logo.png"
+          alt="menu__logo"
+          >
+        </a>
+        <nav class="menu__nav">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#apple" class="menu__link is-active">Apple</a>
+            </li>
+            <li class="menu__item">
```

Comment:
you can add some space between same siblings elements
```suggestion
            </li>
            
            <li class="menu__item">
```